### ISSUE TITLE
Display Image after preprocessing (Python)

### ISSUE THREAD

**Issue by raghavchugh21** [keras-team/keras#14534]
How can I preview or show a sample of the dataset before and after the preprocessing

Preprocessing the Training set
train_datagen = ImageDataGenerator(rescale = 1./255,
shear_range = 0.2,
zoom_range = 0.2,
horizontal_flip = True) #preprocessing_function=thresholding_preprocessing)
training_set = train_datagen.flow_from_directory('dataset/training_set',
target_size = (64, 64),
batch_size = 32,
class_mode = 'binary')

Preprocessing the Test set
test_datagen = ImageDataGenerator(rescale = 1./255)
test_set = test_datagen.flow_from_directory('dataset/test_set',
target_size = (64, 64),
batch_size = 32,
class_mode = 'binary')

**Comment by ymodak** [comment]
https://stackoverflow.com/questions/62217528/how-to-show-results-of-data-augmentation-before-and-after-keras-preprocessing-im
